app/services/geographic/update_ea_area_service.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+module Geographic
```

Comment:
I don't think this belongs to Geographic namespace. This service is not performing any geo mapping tasks, it's just calling other services and updates address area

Reply:
Would "addresses" be better?

Reply:
Yes. It would make sense to reserve Geographic namespace for services that do geo mapping but have no knowledge of business logic of the app. Later on, we could take Geographic into a separate gem and use it across different apps